code/datums/keybindings/carbon.dm

```diff
@@ -0,0 +1,83 @@
+/datum/keybinding/carbon
+	category = KB_CATEGORY_CARBON
+
+/datum/keybinding/carbon/can_use(client/C, mob/M)
+	return iscarbon(M) && ..()
+
+/datum/keybinding/carbon/throw_mode
+	name = "Throw Mode (Toggle)"
+	keys = list("R", "Southwest")
```

Comment:
Can you see if specifying `End` for this works these days?

(For context, the reason `Southwest` equates to `End`, go look at your numpad, assuming you still have one)